specification/src/main/asciidoc/chapters/Sparkplug_10_Conformance.adoc

```diff
@@ -12,20 +12,79 @@ _Sparkplug™ and the Sparkplug™ logo are trademarks of the Eclipse Foundation
 
 [[conformance]]
 == Conformance
- TODO: See Section 7 of http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html
+// TODO: See Section 7 of http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html as a
+// model to use for this chapter
 
-[[conformance_targets]]
-=== Conformance Targets
- TODO: See Section 7 of http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html
+[[conformance_profiles]]
+=== Conformance Profiles
+
+There are four Sparkplug target applications. A Sparkplug infrastructure typically consists of one
+or more of the following application types
+
+* Sparkplug Edge Node
+* Sparkplug Primary Host Application
+* Sparkplug Secondary Host Application
+* MQTT Server
+
+Each application type has specific conformance requirements that must be met. Typically a Sparkplug
+application would only implement one of these profiles. For example an MQTT client wouldn't
+typically be both an Edge Node and a Primary Host Application.
 
 [[conformance_sparkplug_edge_node]]
 ==== Sparkplug Edge Node
- TODO: See Section 7 of http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html
 
-[[conformance_sparkplug_host_application]]
-==== Sparkplug Host Application
- TODO: See Section 7 of http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html
+A Sparkplug Edge Node is typically an 'Edge Gateway'. It sends and receives data to an MQTT Server
+using the spBv1.0/# namespace. Edge Nodes typically interact with physical devices to gather data
+and also write to device outputs.
 
 [[conformance_sparkplug_primary_host_application]]
 ==== Sparkplug Primary Host Application
- TODO: See Section 7 of http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html
\ No newline at end of file
+
+A Sparkplug Primary Host Application it typically at a 'central location' and primarily receives
+data from multiple Sparkplug Edge Nodes. It also may send command messages to Sparkplug Edge Nodes
+to write to outputs at Sparkplug devices. A Primary Host Application also sends rebirth requests to
+Edge Nodes when required.
+[tck-testable tck-id-conformance_primary_host]#[yellow-background]*Primary Host Application MUST
+publish 'STATE' messages that represent its Birth and Death Certificates.*#
+
+[[conformance_sparkplug_secondary_host_application]]
+==== Sparkplug Secondary Host Application
+
+A Sparkplug Secondary Host Application it typically at a 'central location' and primarily receives
+data from multiple Sparkplug Edge Nodes. It also may send command messages to Sparkplug Edge Nodes
+to write to outputs at Sparkplug devices. A Secondary Host Application also sends rebirth requests
+to Edge Nodes when required.
+[tck-testable tck-id-conformance_primary_host]#[yellow-background]*Primary Host Application MUST
```

Comment:
Should it be: 'Secondary Host Application MUST NOT publish 'STATE' messages that represent its Birth and Death Certificates.'